projects/Rockchip/patches/linux/default/linux-1003-for-libreelec.patch

```diff
@@ -726,3 +764,91 @@ index 70ab4fbdc23e..4dbc370e538f 100644
  		cec_transmit_attempt_done(adap, cec->tx_status);
  	}
  	if (cec->rx_done) {
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Matthias Reichl <hias@horus.com>
+Date: Fri, 19 Mar 2021 12:14:17 +0100
+Subject: [PATCH] ALSA: pcm: fix ELD constraints for some compressed audio
+ formats
+
```

Comment:
This should imo be moved to common folder for mainline Linux. It's useful for all platforms...

Reply:
I also thought so, but I'm not sure if it messes something up for generic. would it @HiassofT?

Reply:
No objections moving this to default kernel patches.

Generic uses the Intel HDA ALSA driver, not ASoC, so won't be affected by the change.